.github/workflows/release.yml

```diff
@@ -13,16 +13,31 @@ jobs:
       contents: write
     steps:
       - uses: actions/checkout@v4
-      
+
       - name: Set up Rust
         uses: actions-rs/toolchain@v1
         with:
           profile: minimal
           toolchain: stable
-          
+          override: true
+          components: rustfmt, clippy
+
+      - name: Set up cargo cache
+        uses: actions/cache@v3
```

Comment:
There is also a [dedicated action](https://github.com/Swatinem/rust-cache) that is [a bit](https://github.com/Swatinem/rust-cache/blob/master/src/cleanup.ts#L10) [smarter](https://github.com/Swatinem/rust-cache?tab=readme-ov-file#cache-effectiveness) in what is cached and also handles some cache key edge cases, which reduces the cache blob size and subsequently improves the action performance

Reply:
Thanks, that was much easier to use of course!